testplan/report/testing/schemas.py

```diff
@@ -75,111 +77,60 @@ def _deserialize(self, value, attr, data):
 
 
 class EntriesField(fields.Field):
```

Comment:
At this point it's clear that this class needs to be made much simpler or much more complicated...